nodes/object_operators/ScParent.py

```diff
@@ -10,10 +10,32 @@ class ScParent(Node, ScObjectOperatorNode):
     bl_label = "Parent"
 
     prop_obj: PointerProperty(type=bpy.types.Object, update=ScNode.update_value)
+    ops_type: EnumProperty(items=[('OBJECT', 'Object', ''), ('WITHOUT_INVERSE', 'Object(Without Inverse)', '')], default='OBJECT', update=ScNode.update_value)
```

Comment:
in_ops_type
(Just for consistency in naming variables)

Reply:
Fixed.